### DISCUSSION TITLE
Agenda for the Flatcar Office Hours on Wednesday, 11th of December 2024, 2:30pm UTC

### DISCUSSION THREAD

**Discussion by tormath1** [flatcar/Flatcar#1587] [Flatcar Office Hours]
In the office hours we discuss news and happenings in and around Flatcar, provide a space for our user community to discuss with maintainers, show demos, and do our release planning.

- Jitsi: https://meet.flatcar.org/OfficeHours
- Youtube live stream / recording: https://www.youtube.com/live/7jBb-scSN4k
  - All recordings: https://www.youtube.com/@FlatcarLinux/live
- Community Calls calendar (all future calls): [Google Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [ical](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)
  
## Welcome
- Brief intro/check-in of the new participants joining the call. Please introduce yourself and share what brings you here today.

## Agenda items
- "Showcasing how to get the Nebraska project up and running locally." by @ErvinRacz 
- Add your item in the discussion :) 

### End of support for CGroupsV1 in early 2025
<details><summary>Flatcar will stop supporting the cgroups v1 backwards compatibility mode in Q1 2025.</summary>

- With our adoption of systemd-256, support for legacy CGroupsV1 will end in **Alpha, Beta, and eventually Stable**.
- [Enabling legacy CGroupsV1 during deployment](https://www.flatcar.org/docs/latest/container-runtimes/switching-to-unified-cgroups/#starting-new-nodes-with-legacy-cgroups) will not be supported anymore.
- Nodes that use CGroupsV1 legacy mode will *fail to update*. This ensures your workloads will not be disrupted. Enable CGroupsV2 on your legacy CGroupsV1 nodes in order to successfully update.
- LTS-2024 will support CGroupsV1 until late 2025.
</details>

### Release Planning

- https://github.com/orgs/flatcar/projects/7/views/8

### Administrative tasks after meeting concluded
- [x] Create a new Github discussion for the next Flatcar Office Hours: https://github.com/flatcar/Flatcar/discussions/new?category=flatcar-office-hours
- [x] Close this discussion as "Resolved".

**Comment by ervcz** [comment]
Hello! I'd like to propose a topic/demo for this session: 
**Showcasing how to get the Nebraska project up and running locally.** 

It could be beneficial for users interested in managing Flatcar updates with their own update server instance. 

The demo would include:
 - running Flatcar with qemu locally
 - customize Flatcar so that it connects to the local Nebraska
 - spinning up Nebraska
 - add a new Flatcar release package in Nebraska
 - release the new package on the local Flatcar instance

 Looking forward to hearing thoughts or additional suggestions!
